appcli/models/cli_context.py

```diff
@@ -139,6 +167,14 @@ def get_stack_configuration_file(self) -> Path:
         """
         return self.configuration_dir.joinpath("stack-settings.yml")
 
+    def get_stack_configuration_file_schema(self) -> Path:
+        """Get the location of the configuration file
```

Comment:
pydoc is wrong. , same with the new method above, just forgot to update following the copy/paste